### PR TITLE
Add deployments to runner role.

### PR DESCRIPTION
**Pull request by kevinreiss**

### CODE REVIEW

File: runner.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Runner role
 ---
 # Runner role
 
-The runner role switches off between team members each week.  The runner should be somebody who is planning to be in for the whole week.
+The runner role switches off between team members each week.  The runner should be somebody who is planning to be in for the whole week. The runner as executes a deployment of all our applications to staging and production during the course of their week as the runner.
```
Comment (sandbergja) [inline runner.md:7]: Didn't we say it was the Board Manager who would deploy?  Since the runner already has a lot to do?

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @kevinreiss .  Just one thing I remembered differently

File: runner.md (hunk 1 above)
Comment (maxkadel) [reply runner.md:7]: That's what I remember as well.

Comment (kevinreiss) [reply runner.md]: I moved the docs over to the board handbook and added a section about the Board Manager/Deployments. Let me know how this looks.

File: board.md [hunk 2]
```diff
@@ -32,3 +33,13 @@ When using Epic tickets do not actually assign points to the ticket itself, Epic
 |5 points|2-3 days|
 |8 points|1 week|
 |13 points|Needs to be broken down into smaller tickets|
+
+## The Board Manager Role
+Every sprint a different team member rotates to the role of "board manager". This individual shares the board with the team during stand-ups and planning meetings and leads us through a discussion of the board. The board manager role should be assumed by a Team member who plans to be in the office for entire sprint. The board manager also executes a deployment of all our applications to staging and production during the course of their week as the board manager.
+
+### Deployments as the Board Manager
+
+The runner executes a production and staging deploy of each of our DACS applications.
```
Comment (sandbergja) [inline board.md:42]:
```suggestion
The board manager executes a production and staging deploy of each of our DACS applications. 
```

**Review by sandbergja — CHANGES_REQUESTED** [review]
Thanks, @kevinreiss 🚴🏻 Just found one more mention of the runner.

**Review by christinach — APPROVED** [review]
Thanks @kevinreiss and @sandbergja

**Pull request merged**